packages/performance/src/utils/attributes_utils.ts

```diff
@@ -29,7 +29,6 @@ export enum VisibilityState {
   UNKNOWN = 0,
   VISIBLE = 1,
   HIDDEN = 2,
-  PRERENDER = 3,
   UNLOADED = 4
```

Comment:
Is `UNLOADED` needed ? It's not part of the spec and we don't use it anywhere.

Reply:
We can remove this one as well.

Reply:
Done.